apps/web/src/server/views/app/layouts/app-wide.layout.njk

```diff
@@ -1,3 +1,33 @@
 {% extends "app/layouts/app.layout.njk" %}
 
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
 {% set containerClasses = 'pins-width-container' %}
+
+{% block content %}
```

Comment:
I'm not sure why this change is needed – this extends from app.layout.njk, which is handling all of this anyway?

Reply:
The `app.layout.njk` doesn't handle this, it's only in the derived templates such as `app-two-column.layout.njk`. Perhaps we should move it into the base layout instead?

Reply:
Oh, right... I actually simplified the layouts in my new PR (this app-wide layout no longer exists) and it takes care of what you're doing here, so not sure if we should spend much time on this here. Yours should just work once both are merged and this file is removed.